app/views/comments/_hide_comments_modal.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="hide-comments-modal" class="hidden">
+  <%= form_with url: "/comments/hide", class: "hide-comments-modal__form", id: "hide-comments-modal__form", method: :patch, remote: true, html: { "data-type": "json" } do |f| %>
+    <div class="hide-comments-modal__content">
+      <p class="pb-2">
+        Are you sure you want to hide this comment? It will become hidden in your post, but will still be visible via the comment's permalink.
```

Comment:
(suggestion / not blocking) - I think it would be helpful for the user to link `the comment's permalink` to the comment permalink

Reply:
And this one for the next pr too :)